[32mINFO    [0m integration.ha.test_backups:test_backups.py:796 Syncing credentials for microceph
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  opensearch/0 [executing] maintenance: Backup setup started.
  opensearch/1 [executing] maintenance: Backup setup started.
  opensearch/2 [executing] maintenance: Backup setup started.
  s3-integrator/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  opensearch/0 [idle] active: 
  opensearch/1 [idle] active: 
  opensearch/2 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=186, count=187, last_expected_id=186)
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: POST -- https://10.234.97.28:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: POST -- https://10.234.97.28:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 187, and now, new count: 155
[32mINFO    [0m integration.ha.test_backups:test_backups.py:796 Syncing credentials for aws
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  opensearch/0 [executing] maintenance: Backup setup started.
  opensearch/1 [executing] maintenance: Backup setup started.
  opensearch/2 [executing] maintenance: Backup setup started.
  s3-integrator/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  opensearch/0 [idle] active: 
  opensearch/1 [idle] active: 
  opensearch/2 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=186, count=187, last_expected_id=186)
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: POST -- https://10.234.97.28:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: POST -- https://10.234.97.28:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:367 Calling: GET -- https://10.234.97.28:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 187, and now, new count: 154
[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:13:14Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active      3  opensearch                                 0  no       
s3-integrator                      active      1  s3-integrator             latest/edge     57  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
opensearch/0                 active    idle   2        10.234.97.243   9200/tcp  
opensearch/1                 active    idle   3        10.234.97.16    9200/tcp  
opensearch/2*                active    idle   4        10.234.97.28    9200/tcp  
s3-integrator/0*             active    idle   0        10.234.97.109             
self-signed-certificates/0*  active    idle   1        10.234.97.141             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.234.97.109  juju-16b271-0  ubuntu@22.04      Running
1        started  10.234.97.141  juju-16b271-1  ubuntu@22.04      Running
2        started  10.234.97.243  juju-16b271-2  ubuntu@22.04      Running
3        started  10.234.97.16   juju-16b271-3  ubuntu@22.04      Running
4        started  10.234.97.28   juju-16b271-4  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 02:02:51 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 02:02:51 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 02:02:51 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-0: 02:02:52 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 02:02:52 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 02:02:52 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 02:03:14 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:03:14 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-2: 02:03:14 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 02:03:15 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:03:15 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 02:03:15 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 02:03:16 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 02:03:16 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-1: 02:03:16 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-2: 02:04:32 ERROR unit.opensearch/2.juju-log certificates:4: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-2: 02:04:37 ERROR unit.opensearch/2.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:38 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:39 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:39 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:41 ERROR unit.opensearch/0.juju-log certificates:4: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-2: 02:04:42 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:45 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:46 ERROR unit.opensearch/0.juju-log certificates:4: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:47 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:48 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:48 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:49 ERROR unit.opensearch/0.juju-log certificates:4: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:50 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:04:51 ERROR unit.opensearch/2.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:51 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:52 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:53 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:54 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:55 ERROR unit.opensearch/0.juju-log certificates:4: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:57 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:04:58 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:04:58 ERROR unit.opensearch/1.juju-log certificates:4: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-0: 02:04:59 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:00 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:01 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:01 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:02 ERROR unit.opensearch/0.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:03 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:03 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:03 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:04 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:04 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:05 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:05 ERROR unit.opensearch/1.juju-log certificates:4: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:05 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:06 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:06 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:07 ERROR unit.opensearch/1.juju-log certificates:4: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:07 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:08 ERROR unit.opensearch/0.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:09 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:09 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:09 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:10 ERROR unit.opensearch/0.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:10 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:11 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:11 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:12 ERROR unit.opensearch/0.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:12 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:13 ERROR unit.opensearch/0.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:13 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:13 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:14 ERROR unit.opensearch/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:14 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:15 ERROR unit.opensearch/1.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:05:15 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:16 ERROR unit.opensearch/1.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:17 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:18 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:19 ERROR unit.opensearch/1.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:20 ERROR unit.opensearch/1.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:21 ERROR unit.opensearch/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:22 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:22 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:23 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:24 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:25 ERROR unit.opensearch/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:26 ERROR unit.opensearch/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:27 ERROR unit.opensearch/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:32 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-1: 02:05:33 ERROR unit.opensearch/1.juju-log node-lock-fallback:2: Cannot connect to the OpenSearch server...
unit-opensearch-2: 02:07:35 ERROR unit.opensearch/2.juju-log s3-credentials:5: Failed to setup backup service with state repository s3 is unreachable
unit-opensearch-2: 02:07:53 ERROR unit.opensearch/2.juju-log s3-credentials:5: Failed to setup backup service with state repository s3 is unreachable

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets